paws.common/R/config.R

```diff
@@ -182,14 +182,53 @@ get_instance_metadata <- function(query_path = "") {
   if (trimws(tolower(get_env("AWS_EC2_METADATA_DISABLED"))) %in% c("true", "1")) {
     return(NULL)
   }
-
+  # Get token timeout for IMDSv2 tokens
+  tokentimeout=trimws(tolower(get_env("PAWS_EC2_IMDSV2_TOKEN_TIMEOUT_SECONDS")))
+  if (is.na(as.numeric(tokentimeout))) {
```

Comment:
NIT: this can be simplified to `!nzchar(tokentimeout)`

Reply:
will be anyway removed with the change above